message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -123,7 +123,7 @@
       </entry>
       <entry value="42425" name="MAV_CMD_DO_ACCEPT_MAG_CAL" hasLocation="false" isDestination="false">
         <description>Accept a magnetometer calibration.</description>
-        <param index="1" label="Bitmask" minValue="0" maxValue="255" increment="1">Bitmask of magnetometers that calibration is accepted (0 means all).</param>
+        <param index="1" label="Magnetometers Bitmask" minValue="0" maxValue="255" increment="1">Bitmask of magnetometers that calibration is accepted (0 means all).</param>
```

Comment:
What does accepting a calibration mean?

@WickedShell Should "0 means all" change/does it have same semantics?

Assuming so see https://github.com/ArduPilot/mavlink/pull/93/commits/01835b2f48bb368416a0281b0f7327743c6c6f8f#r271091458 for pattern.

Reply:
Nope, we magically reflect that to 0xFF internally :)

Reply:
Awesome. @amilcarlucas I have no further comments. Up to @peterbarker or others to merge. I have no rights to do so here.